ci/cloudbuild/build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ## Run builds using Google Cloud Build ##
+#
+# This script runs builds using Google Cloud Build. It expects a single
+# argument of the build name to run. Valid build names are the basenames of the
+# scripts in the `builds/` directory. An optional `--distro=<distro>` flag may
+# be specified to indicate which `Dockerfile.<distro>` to run the named build
+# in, and this build is submitted to Cloud Build to run. A distro of "local"
+# (the default) indicates to run the build locally, without docker, using the
+# invoking user's operating system and environment.
+#
+# Usage: ./build.sh [options] <build-name>
+#
+#   Options:
+#     -d|--distro <name>    The distro name to use.
+#
+#   Build names:
+
+set -euo pipefail
+
+source "$(dirname "$0")/../lib/init.sh"
+source module ci/lib/io.sh
+cd "${PROJECT_ROOT}"
+
+# Use getop to parse and normalize all the args.
```

Comment:
s/getop/getopt/

Reply:
Fixed.

> candidly, I do not know if either is more popular, I just noticed that CLion, and VSCode, and I think github treat `.Dockerfile` as an extension

Done. Changed to `<distro>.Dockerfile`